homeassistant/components/august/sensor.py

```diff
@@ -0,0 +1,148 @@
+"""Support for August sensors."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.sensor import DEVICE_CLASS_BATTERY
+from homeassistant.helpers.entity import Entity
+
+from . import DATA_AUGUST
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=5)
+
+
+async def _async_retrieve_device_battery_state(detail):
+    """Get the latest state of the sensor."""
+    if detail is None:
+        return None
+
+    return detail.battery_level
+
+
+async def _async_retrieve_linked_keypad_battery_state(detail):
+    """Get the latest state of the sensor."""
+    if detail is None:
+        return None
+
+    if detail.keypad is None:
+        return None
+
+    battery_level = detail.keypad.battery_level
+
+    _LOGGER.debug("keypad battery level: %s %s", battery_level, battery_level.lower())
+
+    if battery_level.lower() == "full":
```

Comment:
Why convert to lower? can't you just compare to the expected value?

Reply:
I was being a bit too defensive here. Will switch to checking "Full"/"Medium"/"Low"

Reply:
Fixed in branch battery_review_items_and_model